app/views/batch_processes/show.html.erb

```diff
@@ -25,7 +25,7 @@
     <% @batch_process.oids.each do |parent_oid| %>
       <tr>
         <td class="parent_oid"><%= link_to parent_oid, parent_object_path(parent_oid) %></td>
-        <td class="child_count"><%= ParentObject.find(parent_oid).child_object_count.to_s %></td>
+        <td class="child_count"><%= ParentObject.find_by_oid(parent_oid)&.child_object_count || "(deleted)" %></td>
```

Comment:
Why #find_by_oid when presumably #find also works? (since the oid is the ID for the object?) 

I'm going to check this out to see whether I can get it passing CI, so I'll double check that I'm right about that assumption.